src/main/config/config.c

```diff
@@ -218,7 +218,7 @@ static void validateAndFixConfig(void)
 #endif
 
     if (!isSerialConfigValid(serialConfigMutable())) {
-        pgResetFn_serialConfig(serialConfigMutable());
+        PG_RESET(serialConfig);
```

Comment:
(Another bug ?)

Reply:
This is necessary to remove `pgResetFn_serialConfig` prototype from .h file. PG_RESET uses registry to call `pgResetFn_serialConfig(serialConfig);` and was designed to allow explicit config reset (and it does handle reset template too). But in this case both are functionally equivalent.